session/bootstrap.go

```diff
@@ -582,6 +582,10 @@ const (
 		key(state),
       	UNIQUE KEY task_key(task_key)
 	);`
+	// CreateDistFrameworkMeta create a system table that distributed task framework use to store meta information
+	CreateDistFrameworkMeta = `CREATE TABLE IF NOT EXISTS mysql.dist_framework_meta (
+        host VARCHAR(100) NOT NULL PRIMARY KEY,
```

Comment:
how about here add keyspace field? other functional implement could be done later.

Reply:
> how about here add keyspace field? other functional implement could be done later.

Yes, I will add it.